package/package

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+#
+# Package virtualenv-tools into .deb package with fpm.
+#
+# Notes:
+# - This script is designed to run inside a container, so the error handling
+#   is minimal. However, if you have fpm install you can run directly on
+#   your host.
+
+set -euo pipefail
+
+: ${APP_VERSION:?Must be set! (a.k.a. virtualenv_tools application version) See: README}
+: ${PACKAGE_ITERATION:?Must be set! (a.k.a. debian_revision) See: README}
+: ${PACKAGE_MAINTAINER:?Must be set! format: First Last <first.last@prezi.com>}
+
+function main() {
+  fpm \
+    --verbose \
+    --output-type deb \
+    --input-type python \
+    --name python-virtualenv-tools \
+    --maintainer "${PACKAGE_MAINTAINER}" \
+    --prefix /usr/local/ \
+    --description "A set of tools for virtualenv" \
+    --url "http://github.com/prezi/virtualenv-tools" \
+    --version ${APP_VERSION}-${PACKAGE_ITERATION}prezi \
+    --deb-no-default-config-files \
+    --deb-priority 'optional' \
+    --no-python-dependencies \
+    --python-bin /usr/bin/python2.7 \
+    --python-install-lib lib/python2.7/dist-packages \
```

Comment:
It sets where the python libs should be installed. Without this virtualenv-tools throws an exception after install:
Traceback (most recent call last):
  File "/usr/local/bin/virtualenv-tools", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2927, in <module>
    @_call_aside
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2913, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2940, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 635, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 943, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 829, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'virtualenv-tools==1.0' distribution was not found and is required by the application